cmake/emu/qemu.cmake

```diff
@@ -269,6 +269,22 @@ elseif(QEMU_NET_STACK)
   endif()
 endif(QEMU_PIPE_STACK)
 
+if(CONFIG_CAN)
```

Comment:
We need to think does it make sense to move this configuration to the test/sample cmake files.

Reply:
Again, why? This would require everybody wanting to use CAN emulation in QEMU to replicate this in their own CMakeLists.txt (and, from your other comment, replicate the devicetree node as well). 

We have E1000 support in the QEMU x86 board definition and generic support for enabling this in the build system. Why not CAN?

Reply:
I agree with @henrikbrixandersen, i.e. I don't see a good reason to require every app replicate this code.